shigoto_q/tasks/api/serializers.py

```diff
@@ -121,15 +125,37 @@ class Meta:
         ]
 
 
+class TaskImageSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = TaskImage
+        fields = ["full_name", "repo_url", "image_name", "command"]
+
+    def validate(self, attrs):
+        try:
+            image_service = ImageService()
```

Comment:
create a different file, `shigoto_q/tasks/services.py` for example, and add this logic there, keep the serializers clean